index.js

```diff
@@ -15,7 +16,7 @@ export default class ClapprStats extends ContainerPlugin {
   _inc(counter) {this._metrics.counters[counter] += 1}
   _timerHasStarted(timer){return this[`_start${timer}`] !== undefined}
   _start(timer) {this[`_start${timer}`] = this._now()}
-  _stop(timer) {this._metrics.timers[timer] += this._now() - this[`_start${timer}`]}
+  _stop(timer) {this[`_start${timer}`] ? this._metrics.timers[timer] += this._now() - this[`_start${timer}`] : 0}
```

Comment:
While I was developing this plugin I ran in the same situation sometimes and all of them was an error either on Clappr or in this plugin. =/ Do you know in what situation the stop on `watch` timer was called? Because in theory it'd be called only when it already have started =( https://github.com/leandromoreira/clappr-stats/blob/master/index.js#L103-L110